### ISSUE TITLE
Broken on CCL

### ISSUE THREAD

**Issue by notmgsk** [joaotavora/fiasco#28]
Trying to run the README example with (or any fiasco test suite) with CCL results in the following error:

```
> Error: There is no next method for #<STANDARD-METHOD CCL:STREAM-LINE-COLUMN (FIASCO::COLUMN-COUNTING-OUTPUT-STREAM)>
>        args: (#<FIASCO::COLUMN-COUNTING-OUTPUT-STREAM #x30200669BD6D>)
> While executing: #<CCL::STANDARD-KERNEL-METHOD NO-NEXT-METHOD (STANDARD-GENERIC-FUNCTION STANDARD-METHOD)>, in process listener(1).
```

The full backtrace
```
1 > B
 (15193D0) : 0 (FUNCALL #'#<#<CCL::STANDARD-KERNEL-METHOD NO-NEXT-METHOD (STANDARD-GENERIC-FUNCTION STANDARD-METHOD)>> #<STANDARD-GENERIC-FUNCTION CCL:STREAM-LINE-COLUMN #x30200006906F> #<STANDARD-METHOD CCL:STREAM-LINE-COLUMN (FIASCO::COLUMN-COUNTING-OUTPUT-STREAM)> #<FIASCO::COLUMN-COUNTING-OUTPUT-STREAM #x30200669BD6D>) 101
 (15193F8) : 1 (%CALL-NEXT-METHOD (NIL #<STANDARD-METHOD CCL:STREAM-LINE-COLUMN #> . 2765461)) 253
 (1519428) : 2 (FUNCALL #'#<#<STANDARD-METHOD CCL:STREAM-LINE-COLUMN (FIASCO::COLUMN-COUNTING-OUTPUT-STREAM)>> #<FIASCO::COLUMN-COUNTING-OUTPUT-STREAM #x30200669BD6D>) 61
 (1519450) : 3 (%%STANDARD-COMBINED-METHOD-DCODE (#<STANDARD-METHOD CCL:STREAM-LINE-COLUMN #>) 2765461) 701
 (15194B8) : 4 (INITIALIZE-XP #<CCL::XP-STRUCTURE #x3020065077DD> #<FIASCO::COLUMN-COUNTING-OUTPUT-STREAM #x30200669BD6D>) 381
 (15194E0) : 5 (GET-PRETTY-PRINT-STREAM #<FIASCO::COLUMN-COUNTING-OUTPUT-STREAM #x30200669BD6D>) 221
 (1519500) : 6 (DO-XP-PRINTING #<Compiled-function (:INTERNAL FORMAT) (Non-Global)  #x300000A091CF> #<FIASCO::COLUMN-COUNTING-OUTPUT-STREAM #x30200669BD6D> ((0 #\ ))) 45
 (1519528) : 7 (FUNCALL #'#<(:INTERNAL CCL::DO-IT CCL::XP-PRINT)> #<Compiled-function (:INTERNAL FORMAT) (Non-Global)  #x300000A091CF> #<FIASCO::COLUMN-COUNTING-OUTPUT-STREAM #x30200669BD6D> ((0 #\ ))) 53
 (1519550) : 8 (MAYBE-INITIATE-XP-PRINTING #<Compiled-function (:INTERNAL FORMAT) (Non-Global)  #x300000A091CF> #<FIASCO::COLUMN-COUNTING-OUTPUT-STREAM #x30200669BD6D> (0 #\ )) 533
 (15195B8) : 9 (FORMAT #<FIASCO::COLUMN-COUNTING-OUTPUT-STREAM #x30200669BD6D> "~&~v@{~C~:*~}" 0 #\ ) 1781
 (1519648) : 10 (FUNCALL #'#<(:INTERNAL FIASCO::PP FIASCO::PRETTY-RUN-TEST)> "~A (Suite)" FIASCO-SUITES::FIASCO-EXAMPLES) 213
 (1519668) : 11 (PRETTY-RUN-TEST #<test FIASCO-SUITES::FIASCO-EXAMPLES :tests 3> #<Compiled-function (:INTERNAL FIASCO-SUITES::FIASCO-EXAMPLES FIASCO-SUITES::FIASCO-EXAMPLES) (Non-Global)  #x3020065567DF>) 205
 (1519688) : 12 (FUNCALL #'#<(:INTERNAL #:BODY-SYM77146 FIASCO-SUITES::FIASCO-EXAMPLES)>) 453
 (15196D8) : 13 (FIASCO-EXAMPLES) 1885
 (1519848) : 14 (RUN-SUITE-TESTS #<test FIASCO-SUITES::FIASCO-EXAMPLES :tests 3> :VERBOSE NIL :STREAM #<SYNONYM-STREAM to *TERMINAL-IO* #x3020003C57AD> :INTERACTIVE NIL) 637
 (1519978) : 15 (RUN-TESTS #<Package "FIASCO-EXAMPLES"> :DESCRIBE-FAILURES T :VERBOSE NIL :STREAM #<SYNONYM-STREAM to *TERMINAL-IO* #x3020003C57AD> :INTERACTIVE NIL) 805
 (1519A08) : 16 (RUN-PACKAGE-TESTS :PACKAGE #<Package "FIASCO-EXAMPLES"> :PACKAGES (#<Package "FIASCO-EXAMPLES">) :DESCRIBE-FAILURES T :VERBOSE NIL :STREAM #<SYNONYM-STREAM to *TERMINAL-IO* #x3020003C57AD> :INTERACTIVE NIL) 389
 (1519A80) : 17 (CALL-CHECK-REGS RUN-PACKAGE-TESTS) 229
 (1519AB8) : 18 (TOPLEVEL-EVAL (RUN-PACKAGE-TESTS) NIL) 789
 (1519B30) : 19 (READ-LOOP :INPUT-STREAM #<SYNONYM-STREAM to *TERMINAL-IO* #x3020003C590D> :OUTPUT-STREAM #<SYNONYM-STREAM to *TERMINAL-IO* #x3020003C57AD> :BREAK-LEVEL 0 :PROMPT-FUNCTION #<Compiled-function (:INTERNAL CCL::READ-LOOP) (Non-Global)  #x30000058862F>) 2341
 (1519D78) : 20 (RUN-READ-LOOP :BREAK-LEVEL 0) 157
 (1519DA0) : 21 (TOPLEVEL-LOOP) 93
 (1519DB0) : 22 (FUNCALL #'#<(:INTERNAL (CCL:TOPLEVEL-FUNCTION (CCL::LISP-DEVELOPMENT-SYSTEM T)))>) 109
 (1519DD0) : 23 (FUNCALL #'#<(:INTERNAL CCL::MAKE-MCL-LISTENER-PROCESS)>) 661
 (1519E68) : 24 (RUN-PROCESS-INITIAL-FORM #<TTY-LISTENER listener(1) [Active] #x3020003C470D> (#<CCL:COMPILED-LEXICAL-CLOSURE # #x3020003C423F>)) 669
 (1519EF0) : 25 (FUNCALL #'#<(:INTERNAL (CCL::%PROCESS-PRESET-INTERNAL (CCL:PROCESS)))> #<TTY-LISTENER listener(1) [Active] #x3020003C470D> (#<CCL:COMPILED-LEXICAL-CLOSURE # #x3020003C423F>)) 581
 (1519F98) : 26 (FUNCALL #'#<(:INTERNAL CCL::THREAD-MAKE-STARTUP-FUNCTION)>) 277
```

**Comment by joaotavora** [comment]
I'm sorry, I forgot to ACK this bug.  I'll see what I can do this week.  This might be a problem with CCL itself, but even if that is the case, perhaps the fix is to disable the fancy stream functionality in CCL.

**Comment by sabracrolleton** [comment]
I think this is fixed. At least I am not seeing this behavior.

**Comment by appleby** [comment]
Indeed, this looks like it was fixed in 0cc08e798b10298e7a03254034305a77bf7eb063?

**Comment by joaotavora** [comment]
🤔 I may have meant to link this issue instead of #24 in that commit message.  If two people say this is fixed, let's assume it is.  I myself haven't tested, but seems reasonable.
